opentelemetry-sdk/src/opentelemetry/sdk/resources/__init__.py

```diff
@@ -159,7 +161,7 @@ class Resource:
     _schema_url: str
 
     def __init__(
-        self, attributes: Attributes, schema_url: typing.Optional[str] = None
+            self, attributes: Attributes, schema_url: typing.Optional[str] = None
```

Comment:
Please fix your setup to not change indentation

Reply:
Ok